src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import static lotto.utils.validator.Validator.validateEmpty;
+import static lotto.utils.validator.Validator.validateLength;
+import static lotto.utils.validator.Validator.validateNumberRange;
+import static lotto.utils.validator.Validator.validateUniqueValue;
+
+import java.util.List;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validateEmpty(numbers);
+        validateLength(numbers);
+        validateNumberRange(numbers);
+        validateUniqueValue(numbers);
+        this.numbers = numbers;
+    }
+
+    public List<Integer> getNumbers() {
+        return numbers;
+    }
+
+    public int countMatches(List<Integer> winningNumber) {
+        return (int) numbers.stream().filter(winningNumber::contains).count();
```

Comment:
컨벤션에서도 가독성을 위한 스트림 체인의 개행을 허용하고 있습니다.
스트림 체인에 개행을 두면 한 줄마다 하나의 연산을 위치시킬 수 있어 스트림 연산의 단계를 파악하기 훨씬 쉬워져요!
```suggestion
        return (int) numbers.stream()
                .filter(winningNumber::contains)
                .count();
```